core/ev-admin.php

```diff
@@ -1277,10 +1275,14 @@ function admin_custom_columns( $column_name ) {
   function daily_function() {
 
     $options = SP_External_Videos::get_options();
-    $update_hosts = $options['hosts']; // all
-    $update_authors = SP_EV_Admin::get_authors(); // all
```

Comment:
get_authors is used by both this function and the "Update videos from all channels" button in the settings. I'm preparing a fix for get_authors rather than doing this manually.